cli/sysdump.go

```diff
@@ -34,6 +34,10 @@ func newCmdSysdump(hooks sysdump.Hooks) *cobra.Command {
 			if sysdumpOptions.CiliumNamespace == "" && cmd.Flags().Changed("namespace") {
 				sysdumpOptions.CiliumNamespace = namespace
 			}
+			// Honor --helm-release-name global flag in case it is set and --cilium-helm-release-name is not set
+			if sysdumpOptions.CiliumHelmReleaseName == "" && cmd.Flags().Changed("helm-release-name") {
```

Comment:
Could we deprecate `cilium-helm-release-name`?

Reply:
I was wondering where a change in this would break things. IMO, for a manual diagnostic tool I'd have a lower level of caution. But `sysdump` can be called by `connectivity test`, so there's three places to set the helmReleaseName.

Would the process be:

1. Open an issue noting the deprecation.
2. Put a `// TODO: Deprecated by #nnnn on these lines.